packages/ui5-library-writer/templates/common/ui5.yaml

```diff
@@ -0,0 +1,18 @@
+specVersion: '2.2'
+metadata:
+  name: "<%= libraryName %>"
+type: library
+framework:
+  name: <%= framework %>
+  version: <%= frameworkVersion %>
+  libraries:
+    - name: sap.ui.core
+    - name: themelib_sap_belize
+    - name: themelib_sap_fiori_3
+server:
+  customMiddleware:
+  - name: ui5-middleware-livereload
+    afterMiddleware: compression
+    configuration:
+      extraExts: "xml,json,properties"
+      path: ["src", "test"]
```

Comment:
```suggestion
server:
  customMiddleware:
    - name: fiori-tools-appreload
      afterMiddleware: compression
      configuration:
        port: 35729
        path: webapp
        delay: 300
```

Reply:
Updated, changed the path to test to get live reload of Example.html